src/quo2/components/drawers/bottom_actions/view.cljs

```diff
@@ -0,0 +1,48 @@
+(ns quo2.components.drawers.bottom-actions.view
+  (:require [quo2.components.buttons.button.view :as button]
+            [quo2.components.drawers.bottom-actions.style :as style]
+            [quo2.theme :as quo.theme]
+            [quo2.components.markdown.text :as text]
+            [react-native.core :as rn]))
+
+(defn button-press
+  [id]
+  (println "Button " id " Pressed"))
+
+(defn- view-internal
+  "Options:
+
+  :actions - keyword (default nil) - :1-action/:2-actions
+  :description - string (default nil) - Description to display below the title
+  :button-one-label - string (default nil) - Label for the first button
+  :button-two-label - string (default nil) - Label for the second button
+  :button-one-press - fn (default nil) - Function to call when the first button is pressed
+  :button-two-press - fn (default nil) - Function to call when the second button is pressed
+  :theme - :light/:dark
+  :blur? - bool (default false)  - Boolean to set the blur color variant
+  :scroll - bool (default false) - Whether the iOS Home Indicator should be rendered"
+  [{:keys [actions description button-one-label button-two-label
+           button-one-press button-two-press theme
+           blur?
+           scroll?]}]
+  [:<>
+   [rn/view {:style style/buttons-container}
+    (when (= actions :2-actions)
+      [button/button
+       {:size            40
+        :background      (when scroll? blur?)
+        :container-style style/button-container-2-actions
+        :type            :grey
+        :on-press        button-one-press} button-one-label])
+    [button/button
+     {:size            40
+      :container-style style/button-container
+      :type            :primary
```

Comment:
Please make the button `type` as a prop. Because there are instances in the app where we need to use `:type :danger`.